libs/utils/utils.js

```diff
@@ -793,7 +793,13 @@ export async function loadIms() {
       },
       onError: reject,
     };
-    loadScript(`${base}/deps/imslib.min.js`);
+    const params = new URLSearchParams(window.location.search);
+    const useLHControlGroup = params.get('useAlternateImsDomain');
```

Comment:
If you have something against the query param caching idea, we could couple this line with the previous to save some bytes.

Reply:
We already have `PAGE_URL` cached, using that!